src/friendly-iframe-embed.js

```diff
@@ -452,6 +454,144 @@ export class FriendlyIframeEmbed {
       this.visibilityObservable_.fire(this.visible_);
     }
   }
+
+  /**
+   * @return {!HTMLBodyElement}
+   * @visibleForTesting
+   */
+  getBodyElement() {
+    return /** @type {!HTMLBodyElement} */ (
+        (this.iframe.contentDocument || this.iframe.contentWindow.document)
+            .body);
+  }
+
+  /**
+   * @return {!./service/vsync-impl.Vsync}
+   * @visibleForTesting
+   */
+  getVsync() {
+    return vsyncFor(this.win);
```

Comment:
Yeah, some services are kind of painful to stub. Ideally we should have a general intermediate layer for this IMO, but I'm not familiar enough with the service system to provide a suggestion that would be simple to implement.